renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -0,0 +1,604 @@
+#include <stdio.h>
+#include <sys/types.h>
+#include <vector>
+#include "driver/vulkan/vk_debug.h"
+#include "driver/vulkan/vk_replay.h"
+#include "vk_shader_cache.h"
+
+bool isUsageUAV(ResourceUsage usage)
```

Comment:
The term 'UAV' isn't correct for Vulkan, so this function should have a better name. Perhaps `isDirectWrite` to go along with the final variable stored in the pixel history results.

Reply:
Right, makes sense, changed.